mig/shared/functionality/cat.py

```diff
@@ -46,6 +46,18 @@
 from mig.shared.validstring import valid_user_path
 
 
+def _check_sizes(configuration, paths):
+    serve_maxsize = configuration.migserver_server_maxsize
+
+    serve_paths_stat_results = (os.stat(path) for path in paths)
```

Comment:
I wonder if it should wrap in try/except to avoid a hard script crash if the file system errs out on the `stat`.

Reply:
https://github.com/ucphhpc/migrid-sync/pull/131/files#diff-8f9348586dcddc425f3f8faa0164713569db12d4c9111a3c65d281f3ec973ce8R58